src/python-bindings/common2/py_util.cpp

```diff
@@ -294,3 +319,33 @@ py_list_to_vector_of_strings( PyObject * l, std::vector<std::string> & v, const
 
     return 0;
 }
+
+int
+py_list_to_projection( PyObject * l, classad::References & attrs, const char * name ) {
```

Comment:
I'm not sure I would have put this in the utilities; it seems rather special-purpose (although I could be wrong about that).

For consistency, it should be named `py_list_to_classad_references()`.  I guess if it _is_ special-purpose, `convert_py_projection_to_cpp()` would be fine, too.

Reply:
Projections are exposed in newer functions as classad::References,  the vector of strings should be phased out over time.  (the wire form of projection is a stringlist,  it sucks that you can't just supply that in most of our apis)